### PR TITLE
028 Script for requirement [GeneralResultCodes] GENERIC_ERROR watchdog timeout from HMI

### PR DESCRIPTION
**Pull request by maria-mih**
028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua
All common functions used available at #703
Testing coverage available at #643 as well as https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/feature/extended_policy_functionality/test_sets/PAPT_new_param_audioPassThruIcon.txt

@istoimenova Please review

### CODE REVIEW

File: test_scripts/API/PerformAudioPassThru/028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+-- 1.1. PerformAudioPassThru RPC is allowed by policy
+-- 1.2. All params used in PerformAudioPassThru_request are present and within bounds
+-- 1.3. AudioPassThruIcon exists at apps sub-directory of AppStorageFolder (value from ini file)
+--
+-- 2. Performed steps
+-- Send PerformAudioPassThru (audioPassThruIcon, other params)
+-- HMI sends to SDL UI.PerformAudioPassThru (SUCCESS)
+-- HMI does not send to SDL TTS.Speak
+--
+-- Expected result:
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua]: @maria-mih, please update description: 
as SDL -> HMI: UI.PerformAudioPassThru / TTS.Speak
SDL-> mob: PerformAudioPassThru

File: test_scripts/API/PerformAudioPassThru/028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+-- SDL sends UI.PerformAudioPassThru (audioPassThruIcon, other params) to HMI
+-- SDL sends to HMI UI.PerformAudioPassThru
+-- SDL sends to mobile PerformAudioPassThru (GENERIC_ERROR, success:false, info = "TTS.Speak component does not respond")
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua]: @maria-mih, please clarify is '%component-name%' equal to TTS.Speak

File: test_scripts/API/PerformAudioPassThru/028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+    EXPECT_NOTIFICATION("OnHMIStatus",
+      {hmiLevel = "FULL", audioStreamingState = "ATTENUATED", systemContext = "MAIN"},
+      {hmiLevel = "FULL", audioStreamingState = "AUDIBLE", systemContext = "MAIN"})
+    :Times(2)
+  else
+    EXPECT_NOTIFICATION("OnHMIStatus"):Times(0)
+  end
+
+  self.mobileSession:ExpectResponse(CorIdPerformAudioPassThru, {success = false, resultCode = "GENERIC_ERROR", info = "TTS.Speak component does not respond"})
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua]: @maria-mih, please clarify is '%component-name%' equal to TTS.Speak

File: test_scripts/API/PerformAudioPassThru/028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua (hunk 1 above)
Comment (maria-mih) [inline test_scripts/API/PerformAudioPassThru/028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua]: @istoimenova Updated

File: test_scripts/API/PerformAudioPassThru/028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua (hunk 2 above)
Comment (maria-mih) [inline test_scripts/API/PerformAudioPassThru/028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua]: Clarified in `Clarifications about result codes for PerformAudioPassThru` and updated

File: test_scripts/API/PerformAudioPassThru/028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua (hunk 3 above)
Comment (maria-mih) [inline test_scripts/API/PerformAudioPassThru/028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua]: @istoimenova Clarified and updated

**Comment by maria-mih** [comment]
@istoimenova Please check update on commit fe7de48866e954729e10ab38fb5949ac62915cc9

**Pull request merged**